modules/api/src/main/java/org/apache/ignite/schema/ColumnType.java

```diff
@@ -163,6 +166,37 @@ public static DecimalColumnType decimalOf() {
         );
     }
 
+    /**
+     * Timezone-free value representing a time of day in hours, minutes, seconds,
+     * and fractional seconds depending on type precision.
+     *
+     * @param precision Fractional seconds part length. Allowed values are 3/6/9 for millis/micros/nanos.
```

Comment:
perhaps, it would be better to introduce enum for such length

Reply:
I think it's old javadoc and just should be amended. We should support any values between 0 and 9